modules/install/pages/sizing-general.adoc

```diff
@@ -108,6 +108,40 @@ A reliable high-speed network for intra-cluster and inter-cluster communications
 +
 Most deployments can achieve optimal performance with 1 Gbps interconnects, but some may need 10 Gbps.
 
+
+[#guardrails]
+==== Couchbase Server Guardrails
+
+For safe and reliable cluster operations, and to minimize the severity of cluster outages, clusters using Couchbase Server 7.6 or later have guardrails in place.
+These guardrails proactively monitor for specific system thresholds.
+When a cluster meets these thresholds, these guardrails automatically prevent the following cluster operations, which during these conditions can put the cluster at severe risk of outage and downtime:
+
+* Bucket writes
+* Bucket creation
+* Collection creation
+* Topology changes that involve rebalancing
+
+Server-level enforcement thresholds are designed to ensure the optimal and reliable usage of available system resources by various workloads.
+These thresholds are as follows:
+
+[Attributes]
+|===
+|Cluster Metric |Threshold
+
+|Data Service Resident Ratio (Couchstore)
+|\<= 1%
```

Comment:
I think this is fine, but I did wonder if it needs the backslash escape?
and whether `<=` is clear enough without explanation (but should be fine, is used elsewhere in docs-server in a text context, once, and I don't see any counter-indications in the style guide.